From df0c325a425eb2104e97d22a62a7fe978ea6b8c3 Mon Sep 17 00:00:00 2001
Message-Id: <df0c325a425eb2104e97d22a62a7fe978ea6b8c3@dist-git>
From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sun, 1 Sep 2013 09:53:03 +0200
Subject: [PATCH] Fix AM_LDFLAGS typo

(cherry picked from commit fe502de3bcdd76a0d256206111945ca7e4f4388a)

https://bugzilla.redhat.com/show_bug.cgi?id=1242156

Conflicts:
	src/Makefile.am -- context

Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index a6a5600..53d3429 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1185,7 +1185,7 @@ noinst_LTLIBRARIES += libvirt_driver_nwfilter.la
 endif
 libvirt_driver_nwfilter_la_CFLAGS = $(LIBPCAP_CFLAGS) \
 		-I$(top_srcdir)/src/conf $(LIBNL_CFLAGS) $(AM_CFLAGS) $(DBUS_CFLAGS)
-libvirt_driver_nwfilter_la_LDFLAGS = $(LD_AMFLAGS)
+libvirt_driver_nwfilter_la_LDFLAGS = $(AM_LDFLAGS)
 libvirt_driver_nwfilter_la_LIBADD = $(LIBPCAP_LIBS) $(LIBNL_LIBS) $(DBUS_LIBS)
 if WITH_DRIVER_MODULES
 libvirt_driver_nwfilter_la_LIBADD += ../gnulib/lib/libgnu.la
-- 
2.7.0

